target/goalert
# ui: splash screen from index.html

**Description:**
This PR updates the current app-wide suspense logic in place of a splash screen embedded in the `index.html`. This still solves the issue of the app screen flickering in when not logged in (and vice verse with the login screen), but instead shows a gif of the new brand logo with 3 loading dots.

**Other Changes:**
- Branding has been updated for the GoAlert logo app-wide
- Sidebar width slightly decreased to account for new logo sizing

**How to test:**
1. Delete the folder `web/src/build/static`
2. Run `yarn esbuild` to regenerate it (pre-run `yarn` if needed to install the latest deps)
3. Run `make start`
4. To see the logo for longer, open the browser console and navigate to the Network tab
5. Click "No throttling" in the header to open the drop down, then select "Slow 3G"
6. Refresh the page

<details>
<summary>Screenshots:</summary>

![Screenshot 2023-12-06 at 7 31 53 AM](https://github.com/target/goalert/assets/11381794/5c7e4469-2d8d-4ea7-9b18-f813f8fb74a9)

![Screenshot 2023-12-06 at 7 31 41 AM](https://github.com/target/goalert/assets/11381794/87e93e4b-8fdc-41d9-818f-e32d63ccab6d)

![Screenshot 2023-12-06 at 7 32 09 AM](https://github.com/target/goalert/assets/11381794/5fe5892c-bc08-47b0-ae9d-3293ca39307c)

![Screenshot 2023-12-06 at 7 32 35 AM](https://github.com/target/goalert/assets/11381794/1cb9e4e6-a013-4538-bf44-76d66c823a38)

![Screenshot 2023-12-06 at 7 32 48 AM](https://github.com/target/goalert/assets/11381794/47b2f40e-2857-4a41-81fe-229f2b441395)

![splash_logo](https://github.com/target/goalert/assets/11381794/5e892814-a0b1-420c-bf0e-a36454efe3fb)
</details>
